legate/driver/driver.py

```diff
@@ -65,7 +65,14 @@ def cmd(self) -> Command:
         launcher = self.launcher
         system = self.system
 
-        parts = (part(config, system, launcher) for part in CMD_PARTS)
+        if config.user_script is None:
+            # run with console, so remove the cmd_user_script
+            new_cmd_parts = tuple(
+                x for x in CMD_PARTS_LEGION if x != CMD_PARTS_LEGION[-2]
+            )
+        else:
+            new_cmd_parts = CMD_PARTS_LEGION
```

Comment:
This conditional is no longer necessary, correct? `cmd_user_script` is a no-op if `config.user_script` is None, so it can just aways be included unconditionally, I think.

If that is not the case, then then condition needs to be made more robust:
```
x for x in CMD_PARTS_LEGION if x != cmd_user_script
```
Relying on exactly `CMD_PARTS_LEGION[-2]` to be `cmd_user_script` is fragile in case the order or contents of `CMD_PARTS_LEGION` has to change in the future.

Reply:
You are right, the if else is no longer needed, and I have removed it.